src/python/grpcio/commands.py

```diff
@@ -248,53 +248,22 @@ def get_ext_filename(self, ext_name):
         return filename
 
     def build_extensions(self):
-        def compiler_ok_with_extra_std():
```

Comment:
Just to confirm, removing this means clang take `--stdc++17` flags now?

Reply:
This function tells whether a given compiler is okay with having C and C++ std options at once. If this returns false, the build script uses either of them depending on the source (e.g. C std option for C source files and C++ std option for C++ source files). By removing this function, it always uses either of them.

Reply:
For clang specifically, it means that it will remove `--std=c++17` when compiling C sources. It still keeps the option for C++ sources.

Reply:
Thanks for clarification!